doc/guide/install.rst

```diff
@@ -277,12 +277,42 @@ To install CLIMADA Petals, we assume you have already installed CLIMADA Core wit
+|                                                      | PyCharm        |                                                                                                                                                   |                            |
+|                                                      | & many more    |                                                                                                                                                   |                            |
++------------------------------------------------------+----------------+---------------------------------------------------------------------------------------------------------------------------------------------------+----------------------------+
+| Git GUI (Graphical User Interface)                   | GitHub Desktop | - provides an interface which keeps track of the branch you're working on, changes you made etc.                                                  | CLIMADA developers         |
+|                                                      | Gitkraken      | - allows you to commit changes, push to GitHub etc. without having to use command line                                                            |                            |
+|                                                      |                | - the code itself is not written using these applications but with your IDE of choice (see above)                                                 |                            |
++------------------------------------------------------+----------------+---------------------------------------------------------------------------------------------------------------------------------------------------+----------------------------+
+| Continuous Integration (CI) server                   | Jenkins        | - automatically checks code changes in GitHub repositories, e.g. when you create a pull request for the develop branch                            | CLIMADA developers         |
+|                                                      |                | - performs static code analysis using pylint                                                                                                      |                            |
+|                                                      |                | - you don't need to do any installations, this runs automatically (see: :doc:`GitHub Actions CI <github-actions>`)                                |                            |
++------------------------------------------------------+----------------+---------------------------------------------------------------------------------------------------------------------------------------------------+----------------------------+
```

Comment:
I find this table hard to read and confusing. Could we remove it altogether? It does not provide any helpful information at this point:
* If you made it this far, you already installed Conda/Mamba
* The term "IDE" is explained in the Glossary further down
* Jenkins is a server-side application and nothing you run on your computer. The information on Jenkins should be added to the Continuous Integration Guide
* Git GUIs can be recommended, but I would do it in the same fashion as for the IDEs. For example, GitKraken provides a VSCode plugin called Gitlens, which seems to have the same functionality as the standalone app. For VSCode users, we should therefore recommend the plugin, not the app.

Reply:
Maybe this is something to discuss further, but I would not be in favour of removing the table altogether. If you remember, this idea came up at the climada days as a way of taking people who are still new to all of this by the hand and providing **one** overview of what is what. I understand that many of these terms are also mentioned elsewhere, but it is a lot of information in a lot of different places, especially if you are new to this. 
To your specific points:

- Having already installed Conda/Mamba does not necessarily mean you fully understand what they do and how they relate to other programs you are using. Maybe we can add a link to this overview further up in the installation guide
- We can fully merge the Glossary and this section to prevent overlaps while maintaining the information provided in both
- While you don't run Jenkins on your own machine it is yet another name that comes up, where it may not be clear to new developers how it relates to them. Again, I am in favour of having one overview in one place where all of these things are listed next to eachother and not spread out over different guides. I am open to discussing where the appropriate place for such an overview is / changing the title or even overhauling the entire layout but I do think a comprehensive overview is necessary
- I think we may not even have to give a recommendation one way or another, what was important to me here was explaining what these different things do, i.e. the difference between an IDE and a Git GUI on a conceptual level and how they relate to each other, and then give examples for which apps are in which category

Reply:
Agreed!